pom.xml

```diff
@@ -377,21 +372,6 @@
 			<version>2.1.1</version>
 		</dependency>
 		<dependency>
-			<groupId>org.seamless</groupId>
-			<artifactId>seamless-http</artifactId>
-			<version>1.1.1</version>
-		</dependency>
-		<dependency>
-			<groupId>org.seamless</groupId>
```

Comment:
@Nadahar actually it is not used. If you use it in your PR than it should be added with your commit or you can modify this PR.